client/src/components/common/ETGeneralLayout/ETGeneralLayout.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <VApp>
+        <VMain class="main">
+            <Header />
+            <div>
+                <slot></slot>
+            </div>
+        </VMain>
+    </VApp>
+</template>
+
+<script>
+import Header from '@/components/eventType/Header.vue';
+
+export default {
+    name: 'ETGeneralLayout',
```

Comment:
Do not use contractions, this component relates to the public event type. I would rather created `components/public/calendar/` folder and moved all related to your tasks components there. And name it just `GeneralLayout` without contraction